docs/database/README.md

```diff
@@ -91,6 +93,16 @@ db.query "select name, age from contacts order by age desc" do |rs|
 end
 ```
 
+You can also use query parameters:
```

Comment:
Query parameters are already mentioned in more depth a couple lines above. It would probably make sense to move the explanation of query parameters to a separate section and reference that from both `exec` and `query` sections.

Reply:
OK attempted see a147195